config.js

```diff
@@ -334,6 +313,33 @@ exports.updateFromArgv = function() {
       ["minperiodtimeunranked", "minperiodtimeblitzunranked, --minperiodtimeliveunranked and/or --minperiodtimecorrunranked"]
       ]
     deprecatedArgs.forEach(ar => testDeprecated(...ar))
+
+    let familyArray = []; 
+    function familyArrayFromGeneralArg(generalArg) {
+        familyArray = []; // first, we reinitialize it to empty at every start
+                  // then, for example, will be ["komi", "komiranked", "komiunranked"];
+        let arr = [ "", "ranked", "unranked"]; // our source to inject on "komi" for example
+        for (let e of arr) {
+            e = generalArg + e; // example : e = "komi" + "ranked";
+            familyArray.push(e);
+        }
+    }
+
+    familyArrayFromGeneralArg("komi");
```

Comment:
And then this would be deleted.

Reply:
yes, if the function is a defined variable, we would not need this line anymore

but for the next times the function is called,
i think we should not use `let` because familyArray was already defined earlier right ?

for example : 

```let familyArray = familyArrayFromGeneralArg("komi");

// then later in the code, we dont use let again

familyArray = familyArrayFromGeneralArg("boardsize");
```

Reply:
That's right.

A step further would be to not have an intermediate name for the generated array at all:

```
for (let e of familyArrayFromGeneralArg("komi")) {
    ...
}
for (let e of familyArrayFromGeneralArg("boardsize")) {
    ...
}
```